app/views/conference/schedule.xml.haml

```diff
@@ -0,0 +1,35 @@
+%schedule
+  %version= @conference.revision
+  %conference
+    %acronym= @conference.short_title
+    %title= @conference.title
+    %start= @conference.start_date
+    %end= @conference.end_date
+    %days= (@conference.end_date - @conference.start_date).to_i + 1
+    %timeslot_duration= '00:15' #FIXME
```

Comment:
this shouldn't be hard coded. I've seen it in other places as well, so if it's not supposed to be configurable, it should probably be defined as a constant somewhere.